cpp/include/Ice/ProxyFunctions.h

```diff
@@ -30,22 +30,20 @@ namespace Ice
         }
     }
 
-    /// Downcasts a proxy without confirming the target object's type via a remote invocation.
```

Comment:
I think we should keep this comment (but with the better wording). Instead users need to read `checkedCast` first for the necessary context to understand `uncheckedCast`. It's obvious to us, but we're not the target for these doc-comments.